### DISCUSSION TITLE
email no reply

### DISCUSSION THREAD

**Discussion by UnicorNn1** [teaxyz/white-paper#132] [General]
I have been early registrated from your site for Testnet, which will start soon, but I have not received an e-mail to press oke.Some community members are writing about it and all of them received an e-mail reply after registering on the site, but not me. how can I fix this, please give feedback.

**Comment by thomas-borrel** [comment]
Hi @Unicorn1807 - Could you please let us know when you filled the registration form and the type of email account you used (gmail, protonmail, etc...) so we can investigate further.

**Reply by UnicorNn1** [reply]
I registered 1 month ago and registered with gmail

**Comment by rav2525** [comment]
check your spam folders, alternatively you can signup again with other email address of yours.
Gmail is preferable I guess, since I got an instant confirmation.

**Reply by UnicorNn1** [reply]
thanks for the feedback

**Comment by hanriiver** [comment]
You can check spam folders, or google email use
